doc/techref/justification_codes.md

```diff
@@ -0,0 +1,117 @@
+---
+file_format: mystnb
+---
+
+# Justification codes
+
+To adjust the position of plot embellishments, such as scalebars, directional roses,
```

Comment:
I feel the description could be better and more accurage, but I don't have good suggestions yet.

Reply:
Thanks for helping with the formulations 🙂!